content/es/_index.md

```diff
@@ -5,25 +5,43 @@ status: Completed
+Su meta es explicar los conceptos nativos para la nube en un lenguaje claro y simple sin recurrir a cualquier conocimiento técnico previo.
 
 <p><img class="mt-5" src="/images/homepage/stage.jpg" alt="Una mujer y dos hombres presentando información técnica en el estrado"></p>
 
 ## Contribuyendo
 
-Todo mundo esta invitado a sugerir cambios, adiciones, y mejoras a el Glosario Cloud Native. Empleamos un proceso guiado por la comunidad y gobernado por la CNCF para desarrollar y mejorar este léxico compartido. Este Glosario provee una plataforma neutral de vendedores para organizar un vocabulario compartido alrededor de las tecnologías nativas para la nube. Las contribuciones son bienvenidas de todos los participantes que cumplan con el propósito y los estatutos del proyecto.
+Todo mundo esta invitado a sugerir cambios, adiciones, y mejoras a el Glosario Cloud Native.
+Empleamos un proceso guiado por la comunidad y gobernado por la CNCF para desarrollar y mejorar este léxico compartido.
+Este Glosario provee una plataforma neutral de vendedores para organizar un vocabulario compartido alrededor de las tecnologías nativas para la nube.
+Las contribuciones son bienvenidas de todos los participantes que cumplan con el propósito y los estatutos del proyecto.
 
-Cualquier persona que desee hacer una contribución puede enviar un GitHub issue o crear un pull request. Asegúrese de seguir la [Guía de estilo](/es/style-guide/), leer el documento de [Cómo contribuir](/es/contribute/) y unirse al canal de #glossary en el CNCF Slack. También hay un canal #glossary-localizations para aquellos que quieran ayudar a traducir el glosario a su idioma.
+Cualquier persona que desee hacer una contribución puede enviar un GitHub issue o crear un pull request.
+Asegúrese de seguir la [Guía de estilo](/es/style-guide/), leer el documento de [Cómo contribuir](/es/contribute/) y unirse al canal de [#glossary](https://cloud-native.slack.com/archives/C02TX20MQBB) en el CNCF Slack.
+También hay un canal [#glossary-localizations](https://cloud-native.slack.com/archives/C02N2RGFXDF) para aquellos que quieran ayudar a traducir el glosario a su idioma.
 
 ## Reconocimientos
 
-El Glosario Cloud Native fue iniciado por CNCF Marketing
-Committee (Business Value Subcommittee) e incluye
-contribuciones de [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/),
-[Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/), [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), [Seokho Son](https://www.linkedin.com/in/seokho-son/) y muchos contribuidores mas. Para una lista de contribuidores completa, consulte [esta página de GitHub](https://github.com/cncf/glossary/graphs/contributors).
```

Comment:
Hi @krol3 
I think this part from the source document has been removed by this PR. (Acknowledgement to the Glossary maintainers)
But it seems not urgent issue, can be resolved next time.
I thinks you can open another PR to restore the removed part to make up this PR, or I can merge this PR first, upto your team decision.

Reply:
I agree with opening other PR to remove this. I think we can merge this PR first.

Reply:
I created an issue to track this!! Thank you